pkg/agent/cniserver/server.go

```diff
@@ -556,6 +559,15 @@ func (s *CNIServer) cmdDel(_ context.Context, cniConfig *CNIConfig) (*cnipb.CniC
 	}
 	klog.InfoS("Deleted interfaces for container", "container", cniConfig.ContainerId)
 
+	// Check if any SR-IOV device is still attached before removing primary interfaces.
```

Comment:
Why is this important? Is it because deleting the Pod / netns while VFs are still assigned to it could lead to an incomplete clean up?
Could you add that information to the comment?